src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/SinkIslandCheck.java

```diff
@@ -59,7 +58,9 @@ public SinkIslandCheck(final Configuration configuration)
         super(configuration);
         this.treeSize = configurationValue(configuration, "tree.size", TREE_SIZE_DEFAULT,
                 Math::toIntExact);
-
+        this.minimumImportanceHighway = configurationValue(configuration,
+                "highway.importance.minimum", DEFAULT_MINIMUM_IMPORTANCE_HIGHWAY,
```

Comment:
Nit: This should follow the convention of using the key `minimum.highway.type`.

Reply:
@Bentleysb For the FloatingEdgeCheck, I had given the configurable as `minimum.highway.type`  but as per comments changed to `highway.minimum`. Please see the [PR comment](https://github.com/sayas01/atlas-checks/pull/2#pullrequestreview-160420729). Is there a format for such configurable to refer for future enhancements?

Reply:
Very true, I guess I did not look at existing examples before I made the comment on your PR, @sayas01. I based this comment on the current example in atlas checks of InvalidAccessTagCheck. There is no real convention, just what we think is going to clearly identify what the value is used for.